src/main/java/com/psychology/product/auth/UserSignUp.java

```diff
@@ -0,0 +1,28 @@
+package com.psychology.product.auth;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.*;
+import lombok.Data;
+
+@Data
+public class UserSignUp {
```

Comment:
Rename to LoginRequest. Think, maybe better into login use only email and password?

Reply:
And better do it 2 endpoint's like create user and login user